PureLayout/PureLayout/ALView+PureLayout.h

```diff
@@ -118,6 +120,20 @@ __PL_ASSUME_NONNULL_BEGIN
 /** Pins an edge of the view to a given edge of another view with an offset as a maximum or minimum. */
 - (NSLayoutConstraint *)autoPinEdge:(ALEdge)edge toEdge:(ALEdge)toEdge ofView:(ALView *)otherView withOffset:(CGFloat)offset relation:(NSLayoutRelation)relation;
 
+/** Pin a space between self and the other view [self]-space-[otherView], either on the horizontal or vertical axis. */
+- (NSLayoutConstraint *)autoPinSpace:(CGFloat)space followedByView:(ALView *)otherView onAxis:(ALAxis)axis;
+
+/** Pin a space between self and the other view [self]-space-[otherView], either on the horizontal or vertical axis with the space as a maximum or minimum.  */
+- (NSLayoutConstraint *)autoPinSpace:(CGFloat)space followedByView:(ALView *)otherView onAxis:(ALAxis)axis relation:(NSLayoutRelation)relation;
+
+/** Pin trailing to leading with a space H:[self]-space-[otherView] */
+- (NSLayoutConstraint *)autoPinHorizontalSpace:(CGFloat)space followedByView:(ALView *)otherView;
+
+/** Pin bottom to top with a space V:[self]-space-[otherView] */
+- (NSLayoutConstraint *)autoPinVerticalSpace:(CGFloat)space followedByView:(ALView *)otherView;
```

Comment:
These are the most common methods I use in PureLayout. I'd *really* like to get them in with the standard repo so I have one place to grab all my AutoLayout conveniences.

The problem with `secondView.autoPinEdge(.Leading, toEdge: .Trailing, ofView: firstView, withOffset: 20)` is that it always takes a second to pause and remember which view comes first (it is opposite of the visual layout). It is also a lot harder to read. There are a few places you can make a bug there, and you need to mentally check them all when reading the line.

In a dialog window layout I just made, 5 out of 14 calls to PureLayout were horizontal or vertical space constraints. It seems common enough to have top-level methods.